### PR TITLE
add VLQ Singlet

### PR DESCRIPTION
**Pull request by leonidasprado**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#import model sm
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_proc_card.dat]: please remove this commented line

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# set seed 1
+set Nevents_for_max_weigth 500 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+# specify the decay for the final state particles
+
+set max_running_process 1
+
+decay h > ta+ ta-
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat]: Given that the madspin datacard includes the decay, probably is worth adding it to the filename...

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ lhapdf = pdlabel      ! PDF set                      
+ 306000 = lhaid        ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat]: for PDF please follow the recommendations in 
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
i.e. use
$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS = reweight_PDF

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat (hunk 2 above)
Comment (perrozzi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat]: I mean the *higgs* decay

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat]: do you have any idea of which qcut you will use in pythia?
if you put xqcut to 0 and, say, qcut to 20 GeV the filter efficiency will be very low

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat (hunk 2 above)
Comment (leonidasprado) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_madspin_card.dat]: You mean changing ttH_Singlet_LO to something like ttH_HtoTau_Singlet_LO ?
Is this ok?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat (hunk 3 above)
Comment (leonidasprado) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat]: A question about github: After I do the changes (now after it is merged), do I start a new pull request?

Comment (perrozzi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_Singlet_LO/ttH_Singlet_LO_run_card.dat]: new PR is needed

**Pull request merged**